ui/ui-components/components/navigation.tsx

```diff
@@ -173,7 +174,7 @@ export default function Navigation(props) {
           </span>
         </div>
 
-        {navigationMode === "authenticated" && (
+        {navigationMode !== "unauthenticated" && (
```

Comment:
What's the state of the user when they first visit the page before registering?  I think they would be "unauthenticated", but can't access the setup steps until they register

<img width="1370" alt="Screen Shot 2021-06-03 at 5 36 00 PM" src="https://user-images.githubusercontent.com/303226/120728690-50838d00-c492-11eb-82db-b3fae060492b.png">

Reply:
If there's a ever an attempt to load an action in any of the UIs which get an authentication error (which this is), we immediately take the user to the sign in page with a GET param indicating the original page they tried to view which caused the error.  I'm seeing `http://localhost:3000/session/sign-in?nextPage=/` which means that I'm currently unable to see the `/` page.